src/libm/CMakeLists.txt

```diff
@@ -288,6 +290,51 @@ foreach(SIMD ${SLEEF_SUPPORTED_EXTENSIONS})
   endif(COMPILER_SUPPORTS_${SIMD})
 endforeach()
 
+#
+
+if(BUILD_INLINE_HEADERS)
+  file(MAKE_DIRECTORY ${PROJECT_BINARY_DIR}/include/)
+  set(INLINE_HEADER_FILES_GENERATED "")
+
+  if (SED_COMMAND)
+    foreach(SIMD ${SLEEF_SUPPORTED_EXTENSIONS})
+      if(COMPILER_SUPPORTS_${SIMD})
+	string(TOLOWER ${SIMD} SIMDLC)
+
+	set(INLINE_HEADER_FILE ${PROJECT_BINARY_DIR}/include/sleefinline_${SIMDLC}.h)
+	add_custom_command(
+	  OUTPUT ${INLINE_HEADER_FILE} 
+
+	  COMMAND echo Generating sleefinline_${SIMDLC}.h
+	  COMMAND "${CMAKE_C_COMPILER}" ${FLAG_PREPROCESS} ${FLAG_PRESERVE_COMMENTS} ${FLAG_INCLUDE}${PROJECT_SOURCE_DIR}/src/common ${FLAG_INCLUDE}${PROJECT_SOURCE_DIR}/src/arch ${FLAG_INCLUDE}${CMAKE_CURRENT_BINARY_DIR}/include/ ${FLAG_DEFINE}SLEEF_GENHEADER ${FLAG_DEFINE}ENABLE_${SIMD} ${FLAG_DEFINE}DORENAME ${CMAKE_CURRENT_SOURCE_DIR}/sleefsimddp.c > ${CMAKE_CURRENT_BINARY_DIR}/sleef${SIMD}.h.tmp1
```

Comment:
Please break long lines. It is hard to understand what is going on here. Please try not to pass the 80 chars limit.

Reply:
I added comments in this section.